examples/deadline/All-In-AWS-Infrastructure-Basic/ts/bin/app.ts

```diff
@@ -131,19 +125,7 @@ new ComputeTier(app, 'ComputeTier', {
   vpc: network.vpc,
   renderQueue: service.renderQueue,
   workerMachineImage: MachineImage.genericLinux(config.deadlineClientLinuxAmiMap),
-  keyPairName: config.keyPairName,
+  keyPairName: config.keyPairName ? config.keyPairName : undefined,
```

Comment:
what the point to change this line like this?

Reply:
I think I probably just noticed that the ` ? config.keyPairName : undefined` part of the line doesn't actually do anything since the value is optional and will resolve to undefined anyways without the need for the ternary operator, so I deleted it. It should probably be removed in mainline, but it isn't a big deal.